src/main/resources/templates/db_scripts/schema.json

```diff
@@ -788,6 +788,12 @@
 
       ],
       "uniqueIndex": [
+        {
+          "fieldName": "barcode",
+          "tOps": "ADD",
+          "caseSensitive": false,
+          "removeAccents": false
```

Comment:
The values for these parameters are different to the existing indexes, is that intentional? 

My understanding was that database indexes won't be used unless these parameters match up with how the CQL is interpreted, which would suggest that either the old indexes weren't being used or the new one won't. What have I missed / misunderstood?

Reply:
This is intentional, see discussion on https://issues.folio.org/browse/MODINVSTOR-523

RMB automatically uses the case and accents settings for both creating the indexes and when running a CQL query. This ensures that the indexes are always been used. Please suggest a better wording for the documentation if it is unclear: https://github.com/folio-org/raml-module-builder#cql-relations

Reply:
> This is intentional, see discussion on https://issues.folio.org/browse/MODINVSTOR-523

Thank you for clarifying that and helping me change my out of date understanding.
 
> RMB automatically uses the case and accents settings for both creating the indexes and when running a CQL query. This ensures that the indexes are always been used. Please suggest a better wording for the documentation if it is unclear: https://github.com/folio-org/raml-module-builder#cql-relations

I was not aware of that